cranelift/entity/src/list.rs

```diff
@@ -387,14 +397,47 @@ impl<T: EntityRef + ReservedValue> EntityList<T> {
         &mut pool.data[block + 1..block + 1 + new_len]
     }
 
+    /// Constructs a list from an iterator.
+    pub fn from_iter<I>(elements: I, pool: &mut ListPool<T>) -> Self
+    where
+        I: IntoIterator<Item = T>,
+    {
+        let iterator = elements.into_iter();
```

Comment:
Could this function be implemented as `let mut list = Self::new(); list.extend(elements, pool); list`, to avoid some code duplication?

Reply:
Done.